### PR TITLE
Turn exercise 027 to practice

### PR DESCRIPTION
**Pull request by Ozyugoo**

### CODE REVIEW

**Comment by Ozyugoo** [comment]
Hi, @cuihtlauac completed this and tested a few different arguments on it.

File: practice/027/ex.ml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let v = "group" >::: [
+    "nil" >:: (fun _ -> assert_equal [[]] (Tested.group [] []));
+    "cons" >:: (fun _ -> assert_equal [[["a"; "b"]; ["c"]]; [["a"; "c"]; ["b"]]; [["b"; "c"]; ["a"]];
+    [["a"; "b"]; ["d"]]; [["a"; "c"]; ["d"]]; [["b"; "c"]; ["d"]];
+    [["a"; "d"]; ["b"]]; [["b"; "d"]; ["a"]]; [["a"; "d"]; ["c"]];
+    [["b"; "d"]; ["c"]]; [["c"; "d"]; ["a"]]; [["c"; "d"]; ["b"]]] (Tested.group ["a"; "b"; "c"; "d"] [2; 1]));
+  ]
```
Comment (cuihtlauac) [inline practice/027/ex.ml]: Find better names

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Find

File: practice/027/ex.ml (hunk 1 above)
Comment (Ozyugoo) [reply practice/027/ex.ml]: Please check these as well.

**Review by cuihtlauac — APPROVED** [review]
Thanks @Ozyugoo

**Pull request merged**